internal/storage/fake/testing/bucket_tests.go

```diff
@@ -425,6 +425,31 @@ func (t *bucketTest) matchesStartTime(start time.Time) Matcher {
 	return timeutil.TimeNear(start, slop)
 }
 
+// Given gcs.minObject and gcs.extendedObjectAttributes , assert on attributes of gcs.Object
+func (t *bucketTest) assertOnObjectAttributes(minObj *gcs.MinObject, extendedAttr *gcs.ExtendedObjectAttributes, o *gcs.Object) {
```

Comment:
rename this to expectedMinObj and expectedExtendedAttr

Reply:
Done.